wp-content/themes/ipbs-largo/less/widgets.less

```diff
@@ -9,4 +9,61 @@
         margin-bottom: @measure * 2;
         color: @blue;
     }
+}
+
+// member stations widget
+.widget.ipbs-member-stations {
+    margin-bottom: @measure * 2;
+    ul {
+        @media (min-width: 1000px) {
+            display: grid;
+            grid-template-columns: repeat(3, 1fr);
+        }
+        li {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            border-bottom: 1px solid @grey;
+            padding-top: @measure;
+            padding-bottom: @measure;
```

Comment:
With neither padding-right nor padding-left, around 1000px-1024px, the borders get cramped. If we decrease the image max-width to 100px, and put a left/right padding on these cells, then I think this looks better.

But that should not block merging of this PR.